packages/fuelchain/src/utils/modules.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable consistent-return */
+/* eslint-disable global-require */
```

Comment:
if we're ok in getting more and more different from `typechain` code, would be nice to fix this anti-patterns pointed by eslint

Reply:
The idea is to change the least amount of code possible. In the near future, we can also address and improve the code on the feature to improve CLI. This could also result in a completely different code, as we can use the opportunity to remove `typechain-target-*` and remove dynamic codes. https://github.com/FuelLabs/fuels-ts/issues/480

Reply:
ok makes sense, we can work in refactor it in a different issue. if we're stable at this point I don't see reasons to keep code similar